fex2/preconfigured/experiments/common.sh

```diff
@@ -91,16 +110,28 @@ function header_pop() {
     unset 'execution_header[${top[@]: -1}]'
 }
 
+function is() {
+    local value=$1
+    if [[ "$value" == true ]]; then
+        return 0;
+    elif [[ "$value" == false ]]; then
+        return 1;
+    else
+        error_exit_with_backtrace "Invalid boolean value $value" 1
+    fi;
+}
+
 function read_benchmark_arguments() {
     local input_type=$1
+    declare -n out_input=$2
```

Comment:
out_input is confusing. Rename to something more descriptive

Reply:
I made this similar to [C#](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/out-parameter-modifier). What name would you suggest? Maybe we name it just `input` and add a little comment `# return value` :)

Reply:
Sure